ui/src/components/Rules/Rule/RuleCreator/RuleInput/index.tsx

```diff
@@ -362,201 +355,205 @@ const RuleInput = (props: IRuleInput) => {
+            {possibilities.map((action) => (
+              <option key={action} value={action}>
+                {RulePossibilityTranslations[action]}
+              </option>
+            ))}
+          </select>
         </div>
-      </div>
 
-      <div className="form-row">
-        <label htmlFor="second_val" className="text-label">
-          Second value
-        </label>
-        <div className="form-input">
-          <div className="form-input-field">
-            <select
-              name="second_val"
-              id="second_val"
-              onChange={updateSecondValue}
-              value={secondVal}
-            >
-              <option value={undefined}> </option>
-              <optgroup label="Custom values">
-                {ruleType === RuleType.DATE ? (
-                  <>
-                    <option value={CustomParams.CUSTOM_DAYS}>
-                      Amount of days
-                    </option>
-                    {action != null &&
-                    action !== RulePossibility.IN_LAST &&
-                    action !== RulePossibility.IN_NEXT ? (
-                      <option value={CustomParams.CUSTOM_DATE}>
-                        Specific date
-                      </option>
-                    ) : undefined}
-                  </>
-                ) : undefined}
-                {ruleType === RuleType.NUMBER ? (
-                  <option value={CustomParams.CUSTOM_NUMBER}>Number</option>
-                ) : undefined}
-                {ruleType === RuleType.BOOL ? (
-                  <option value={CustomParams.CUSTOM_BOOLEAN}>Boolean</option>
-                ) : undefined}
-                {ruleType === RuleType.TEXT ? (
-                  <option value={CustomParams.CUSTOM_TEXT}>Text</option>
-                ) : undefined}
-                <MaybeTextListOptions ruleType={ruleType} action={action} />
-              </optgroup>
-              {ConstantsCtx.constants.applications?.map((app) => {
-                return (app.mediaType === MediaType.BOTH ||
-                  props.mediaType === app.mediaType) &&
-                  action != null &&
+        {/* Second Value Selection */}
+        <div>
+          <label
+            htmlFor="second_val"
+            className="mb-1 block text-sm font-medium"
+          >
+            Second Value
+          </label>
+          <select
+            name="second_val"
+            id="second_val"
+            onChange={updateSecondValue}
+            value={secondVal}
+            className="w-full rounded-lg p-2 text-zinc-100 focus:border-amber-500 focus:ring-amber-500"
+          >
+            <option value={undefined}> </option>
+            <optgroup label="Custom values">
+              {ruleType === RuleType.DATE ? (
+                <>
+                  <option value={CustomParams.CUSTOM_DAYS}>
+                    Amount of days
+                  </option>
+                  {action &&
```

Comment:
This must've been a merge conflict as it was recently changed from this to fix a bug

```suggestion
                  {{action != null &&
```

Reply:
I confirmed that there is only one opening `{`